rune/spec.go

```diff
@@ -83,6 +83,9 @@ created by an unprivileged user.
 		}
 		spec := specconv.Example()
 		spec.Process.Cwd = "/var/run/rune"
+		spec.Root = &specs.Root{
+			Path: "rootfs",
```

Comment:
I don't believe anybody can understand why this change is related to readonly rootfs. You need to **EXPLICITLY** set readonly as false. Otherwise, you need to add a comment. Let code say what it does.